foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/ExtensionsSecurityProjectsMigration.java

```diff
@@ -0,0 +1,92 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.Map;
+
+import static org.technologybrewery.baton.util.FileUtils.replaceLiteralInFile;
+
+/**
+ Baton migration class to verify whether a migration is required for those java modules that the aiops reference was
+ replaced with aissmeble.
+ **/
```

Comment:
Would change this comment a little. It's not just verifying whether a migration is required. It also does the migration when necessary.

Reply:
Reworded the comment to better match the efforts of the migration.